docs/sources/panels/visualizations/time-series/_index.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Time series panel"
+keywords = ["grafana", "graph panel", "time series panel", "documentation", "guide", "graph"]
+weight = 1200
++++
+
+# Time series panel
+
+> **Note:** This is a beta feature. Time series panel is going to replace the Graph panel in the future releases.
+
+Time series panel is a robust visualization to plot time series data. It can render as a line, a path of dots, or a series of bars. This type of graph is versatile enough to display almost any time-series data.
+
+For Time series panel examples, refer to the Grafana Play dashboard [New Features in v7.4](https://play.grafana.org/d/nP8rcffGk/new-features-in-v7-4?orgId=1).
+
+## Data and field options
+
+Time series visualizations allow you to apply:
+
+- [Data transformations]({{< relref "../../transformations/_index.md" >}})
+- [Field options and overrides]({{< relref "../../field-options/_index.md" >}})
+- [Thresholds]({{< relref "../../thresholds.md" >}})
+
+You can also use field options to create different types of graphs or adjust your axes:
+
+- [Create line graphs]({{< relref "create-line-graphs.md" >}})
+- [Create bar graphs]({{< relref "create-bar-graphs.md" >}})
```

Comment:
bar charts are typically category charts, very rarely are they timeseries. not sure what best to call this, it's a bar style mode, dunno. it's a tough call.

we have a new visualization for bar charts soon to be alpha: https://github.com/grafana/grafana/pull/30323

Reply:
Correct. This topic is how to show time series data in a bar chart. I am open to other suggestions, but we need to consider what the user will want to do. Right now, they use the Graph panel to make bar charts. In future, we will have a better tool for them, but this is what we've got now.